mods/ra/rules/defaults.yaml

```diff
@@ -360,6 +360,10 @@
 	AttackMove:
 	Passenger:
 		CargoType: Infantry
+		CargoCondition: noexperience
```

Comment:
Can we please change this to `disable-experience`? The naming convention for conditions is to use dashes between words.